---
title: Methodology for biomarker discovery with reproducibility in microbiome data
  using machine learning
date: '2024-01-01'
draft: true
publishDate: '2024-12-17T17:20:34.653475Z'
authors:
- David Rojas-Velazquez
- Sarah Kidwai
- Aletta D. Kraneveld
- Alberto Tonda
- Daniel Oberski
- Johan Garssen
- Alejandro Lopez-Rincon
publication_types:
- '2'
abstract: 'Background: In recent years, human microbiome studies have received increasing
  attention as this field is considered a potential source for clinical applications.
  With the advancements in omics technologies and AI, research focused on the discovery
  for potential biomarkers in the human microbiome using machine learning tools has
  produced positive outcomes. Despite the promising results, several issues can still
  be found in these studies such as datasets with small number of samples, inconsistent
  results, lack of uniform processing and methodologies, and other additional factors
  lead to lack of reproducibility in biomedical research. In this work, we propose
  a methodology that combines the DADA2 pipeline for 16s rRNA sequences processing
  and the Recursive Ensemble Feature Selection (REFS) in multiple datasets to increase
  reproducibility and obtain robust and reliable results in biomedical research. Results:
  Three experiments were performed analyzing microbiome data from patients/cases in
  Inflammatory Bowel Disease (IBD), Autism Spectrum Disorder (ASD), and Type 2 Diabetes
  (T2D). In each experiment, we found a biomarker signature in one dataset and applied
  to 2 other as further validation. The effectiveness of the proposed methodology
  was compared with other feature selection methods such as K-Best with F-score and
  random selection as a base line. The Area Under the Curve (AUC) was employed as
  a measure of diagnostic accuracy and used as a metric for comparing the results
  of the proposed methodology with other feature selection methods. Additionally,
  we use the Matthews Correlation Coefficient (MCC) as a metric to evaluate the performance
  of the methodology as well as for comparison with other feature selection methods.
  Conclusions: We developed a methodology for reproducible biomarker discovery for
  16s rRNA microbiome sequence analysis, addressing the issues related with data dimensionality,
  inconsistent results and validation across independent datasets. The findings from
  the three experiments, across 9 different datasets, show that the proposed methodology
  achieved higher accuracy compared to other feature selection methods. This methodology
  is a first approach to increase reproducibility, to provide robust and reliable
  results.'
featured: false
publication: '*BMC Bioinformatics*'
doi: 10.1186/s12859-024-05639-3
links:
- name: URL
  url: http://dx.doi.org/10.1186/s12859-024-05639-3
---

